### DISCUSSION TITLE
Meta Classification Proposal

### DISCUSSION THREAD

**Discussion by utaninja** [TripSit/drugs#114] [Classification]
Hi there!

In the current combination chart, there are two types. One is a specific drug (eg: alcohol) and the other is a drug type (eg: opioids). This causes numerous issues and is not as clear as the chart could be. I would like to propose for the current combination chart a meta-change for all drug categories into drug-specific drugs.

For example - in the opioids category. Included in this category is methadone, which has concerns of serotonin syndrome with some interactions. However, drugs such as oxycodone or heroin which are also in this category, do not have this concern. Another example of 4mmc, a drug that is considered an amphetamine but also has serotonin syndrome risk. However, drugs such as amphetamine or dextroamphetamine do not have this concern.

I believe that the chart in the past did not factor in distinguishing factors of different drugs within the categories, and other drugs in the category become more common in use. It is clear some consideration was done (such as tramadol being there in addition to opioids) but furthering this would be good.

Part of the help with the future dynamic chart and expanded database will help with distinguishing interactions between drugs in the same drug class.

The current drug groups that we have excluded MAOIs (which I discuss below) are the following-
2C-T-x, 2C-x, 5-MeO-xxT, Amphetamines, Benzodiazepines, DOx, NBOMes, Opioids, SSRIs

It seems that the best course of action would be to replace these with the most commonly used recreational drug (/our best guess for this) for the interaction. As an example, we would replace benzodiazepines with alprazolam. As far as I can tell, nearly all of the interactions if replaced with the drug specific names would still apply as the research seems to have been done for the "main/popular" drugs in that category.

MAOIs I'm unsure what best to do for that would be. We could expand the interactions to separate MAO-A, MAO-B, and RIMAs. But this of course would need interactions researched for all of these. We do have some distinguishing factors for interactions already in some combinations.
(such as
2C-T-x + MAOIs =  Caution
MAO-B inhibitors can increase the potency and duration of phenethylamines unpredictably, which could be dangerous given the unpredictability of the 2C-T-x series)
I would be okay for now with leaving MAOIs as in, preferably also continuing to expand interactions for it.

I'd love to get input for this and if this seems good, I'll post a poll/suggestion discussing which the combinations could be changed over to and we can get that changed over :D - it will also be a big help for the interactions research as to narrow our scope and be more specific

- A concern that came to me as writing this up as it may narrow the scope of interactions for the current database. Perhaps we could also keep the interactions for the categories in the database for now and adding a note mentioning that this is in general for the category and it may change depending on specific drug.. etc.

There was some discussion on the discord server and I'm forwarding most of the messages here (leaving out some small comments to keep things concise)

**Comment by zelixir25** [comment]
I think this might cause even more confusion. People using the combo chart might see, for example, that we only list sertraline (or whatever) instead of SSRIs and be unsure if it also applies to citalopram or another SSRI. IMO it's better to have some kind of note on the chart that the combinations are generalized and that individual drugs within classes might pose more/less of a risk, do your own research, etc.

**Comment by utaninja** [comment]
WorriedHobbiton:
I’m a little confused about the proposals direction – are we considering moving from a more classification and substance specific based interaction system to focusing on an individual substance within each class for identifying potential interactions?

If so, definitely see the value in aiming for a more specific, substance focused approach. It seems like it could clarify some of the information and make it more straightforward and accessible.

However, my only concern is this specificity might narrow our focus too much. Focusing on a single representative substance like alprazolam for benzodiazepines or methadone for opioids, while it still captures a large part of associated risk, there’s a chance of unintentionally overlooking important interactions specific to other substances within the same classification. 

For example by zeroing in on alprazolam, we might skip over how diazepam’s interaction with CYP2C19 affects its potency and duration. And with opioids, spotlighting methadone for its serotonin syndrome risk (which is important), could lead us to overlook how codeine’s interaction with CYP2D6 plays a big part in its effectiveness.  In the amphetamine category focusing only on amphetamine might cause us to miss the distinct serotonergic effects of methamphetamine and the unique risk profile of MDMA.

While picking out the popular substances like alprazolam or methadone makes things a bit neater and streamline information, we risk missing the full story on how these individual substances interact with our bodies and other substances. 

Maybe there’s a way to keep things simple for a quick look to keep things practical and accessible but also offer the chance to dive deeper into each drug’s unique interactions for those who need more info, or perhaps choosing 2-3 representatives that might better capture the nuances and diversity of pharmacological interactions.

I’m fully behind the idea of refining our approach to ensure it serves our community’s needs best.

**Reply by utaninja** [reply]
@LunaUrsa Yeah your impression is right: We want to move away from using categories like 'benzos" and label each drug individually. I get what you're saying about how there might be speficics for each drug that will need to be talked out, but i think if we can all agree on the basics then it's better than having no combo info at all for each of those categorized substances
How it is now: If you check any benzo there is no explicit combo info, even though "any benzo" + alcohol is likely dangrous

**Reply by utaninja** [reply]
WorriedHobbiton: Yeah and I hear that, it’s a wide enough net being cast that you’ll catch most of the fish, which is great. It was just a thought that came to mind, but my mind tends to go to the ones who may fall through the cracks. But that’s also what we are here for. To provide additional information/education on the stuff we unfortunately can’t easily fit or implement into the chart/database.

**Comment by LunaUrsa** [comment]
I made a pull request of what this would look like in practice here: https://github.com/TripSit/drugs/pull/127

**Comment by utaninja** [comment]
This post is in regard to the current printout that says "opioids" for example. Discussion about the database, dynamic chart, etc can be seen here: https://github.com/TripSit/drugs/pull/127

> I think this might cause even more confusion. People using the combo chart might see, for example, that we only list sertraline (or whatever) instead of SSRIs and be unsure if it also applies to citalopram or another SSRI. IMO it's better to have some kind of note on the chart that the combinations are generalized and that individual drugs within classes might pose more/less of a risk, do your own research, etc.

@zelixir25 do you think the benefit of this outweighs the cons of using drug categories? I feel conflicted as I'm concerned some may get into harm by thinking that the drug is fine to mix as it's part of a certain drug class. Especially as it gets to more obscure drugs. It seems that this is sort of a picking the least bad option here.

> For example by zeroing in on alprazolam, we might skip over how diazepam’s interaction with CYP2C19 affects its potency and duration. And with opioids, spotlighting methadone for its serotonin syndrome risk (which is important), could lead us to overlook how codeine’s interaction with CYP2D6 plays a big part in its effectiveness. In the amphetamine category focusing only on amphetamine might cause us to miss the distinct serotonergic effects of methamphetamine and the unique risk profile of MDMA.

This is very true and a good point, however, we currently have this issue just reversed. By giving generalizations for opioids, it overlooks certain specific interactions, such as methadone's serotonergic activity.

> Maybe there’s a way to keep things simple for a quick look to keep things practical and accessible but also offer the chance to dive deeper into each drug’s unique interactions for those who need more info, or perhaps choosing 2-3 representatives that might better capture the nuances and diversity of pharmacological interactions.

This is what I am leaning towards, but I don't think we should triple some of the categories for the main chart.

What if we gave a specific drug and then we add a footnote mentioning many interactions for it applies for other drugs in its categories, with some exceptions and some others interacting more than the specific drug mentioned before.

I'm very open to suggestions and don't see a perfect answer to this.
